lib/ansible/plugins/connection/napalm.py

```diff
@@ -155,10 +155,8 @@ def __init__(self, play_context, new_stdin, *args, **kwargs):
 
     def _connect(self):
         if not HAS_NAPALM:
-            raise AnsibleError(
-                'Napalm is required to use the napalm connection type.\n'
-                'Please run pip install napalm'
-            )
+            raise AnsibleError('The "napalm" python library is required to use the napalm connection type.\n')
```

Comment:
Could you use `missing_required_lib()` from `lib/ansible/module_utils/basic.py` to generate this error message?

Reply:
I don't think so.  As of now `missing_required_lib` has specific wording around modules.